Cargo.toml

```diff
@@ -75,6 +75,7 @@ dashmap = { version = "5.5.3" }
 data-encoding = { version = "2.5.0" }
 derivative = { version = "2.2.0" }
 directories = { version = "5.0.1" }
+dirs-sys = { version = "0.4.1" }
```

Comment:
This is already a dependency because it powers `directories` under-the-hood, but exposes the low-level methods we need to use the custom XDG behavior.